furms-database-module/src/main/java/io/imunity/furms/db/sites/SiteDatabaseRepository.java

```diff
@@ -83,6 +93,14 @@ public boolean exists(String id) {
 		return repository.existsById(fromString(id));
 	}
 
+	@Override
+	public boolean existsByShortId(String shortId) {
+		if (isEmpty(shortId)) {
```

Comment:
That looks wrong - why this method treats empty shortId as non existing? I would prefer not to have any if there, or to have precondition failing on illegal argument